tests/by-util/test_du.rs

```diff
@@ -935,3 +934,61 @@ fn test_du_symlink_multiple_fail() {
     assert_eq!(result.code(), 1);
     result.stdout_contains("4\tfile1\n");
 }
+
+#[test]
+// Disable on Windows because of different path separators and handling of null characters
+#[cfg(not(target_os = "windows"))]
+fn test_du_files0_from() {
+    let ts = TestScenario::new(util_name!());
+    let at = &ts.fixtures;
+
+    let mut file1 = at.make_file("testfile1");
+    file1.write_all(b"content1").unwrap();
```

Comment:
What's the reason for creating the files in this way instead of using `at.write("testfile1", "content1");`?

Reply:
no good reason, i followed the example above